SPECS/vim/vim.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "vim-8.2.5154.tar.gz": "43bc45d305cf93fb4ba8f6619c7ac02209daee1f8f3ae071c2dbd635d9f587e3"
+  "vim-8.2.5154.tar.gz": "366a45168f59d7218ea22597432e64ac05c2626703b5d8fa70f37bfd674cd2c1"
```

Comment:
Filename here is incorrect

Reply:
Had my build environment fall over, forgot to redo this change. Thanks for catching this one.